### ISSUE TITLE
[caffe2] AffineChannelOp

### ISSUE THREAD

**Issue by edubois** [pytorch/pytorch#8730]
Hi,

If I use a SpatialBN (spatial batch normalization), when using the net in production, do I need to put a AffineChannelOp or is setting the flag is_test to true enough?

What do you advise? If I need to use an AffinChannelOp, how to do this?

Thanks a lot for anybody helping me.
